deploy/gke.md

```diff
@@ -0,0 +1,209 @@
+This is a known issue in RBAC in Kubernetes and Kubernetes Engine versions 1.6 and later. (@see https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control).
+
+A fast workarround:
+
+```
+echo -n "
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: cluster-admin-user
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: User
+  name: <the current user using kubectl ; usually the google account>
+  namespace: kube-system" | kubectl apply -f -
+```
+
+4. Deploy KONG
+
+You can now deploy kong 
+```
+kubectl apply -f namespace.yaml &&
+kubectl apply -f custom-types.yaml &&
+kubectl apply -f postgres.yaml &&
+kubectl apply -f rbac.yaml &&
+kubectl apply -f ingress-controller.yaml &&
+kubectl apply -f kong.yaml  &&
+kubectl apply -f gke-kong-proxy-loadbalancer-service.yaml 
+```
+
+Should display:
+
+```
+namespace "kong" created
+customresourcedefinition "kongplugins.configuration.konghq.com" configured
+customresourcedefinition "kongconsumers.configuration.konghq.com" configured
+customresourcedefinition "kongcredentials.configuration.konghq.com" configured
+customresourcedefinition "kongingresses.configuration.konghq.com" configured
+service "postgres" created
+statefulset "postgres" created
+serviceaccount "kong-serviceaccount" created
+clusterrole "kong-ingress-clusterrole" configured
+role "kong-ingress-role" created
+rolebinding "kong-ingress-role-nisa-binding" created
+clusterrolebinding "kong-ingress-clusterrole-nisa-binding" configured
+service "kong-ingress-controller" created
+deployment "kong-ingress-controller" created
+deployment "kong" created
+service "kong-proxy" created
+```
+
+You can now retrieve the associated IP for kong-proxy (or you can use  directly your static IP if you used one):
+
+`kubectl get services -n kong` should display :
+```
+NAME                      TYPE           CLUSTER-IP      EXTERNAL-IP    PORT(S)     
+kong-ingress-controller   ClusterIP      10.42.42.1   <none>         8001/TCP    
+kong-proxy                LoadBalancer   10.42.42.2   35.204.42.1   80:30095/TCP,443:31166/TCP   
+postgres                  ClusterIP      10.42.42.3   <none>         5432/TCP                    
+```
+
+Now, 
+
+```
+curl 35.204.42.1
+```
+
+should display:
+
+```
+{"message":"no route and no API found with those values"}
+```
+
+5. Test Your deployment
+
+  - Deploy a dummy application : `kubectl create namespace dummy && curl https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/manifests/dummy-application.yaml -n dummy`
+  - Add an ingress:
+
+```
+echo -n "
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: dummy
+  namespace:  dummy
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  rules:
+    - host: dummy.dummy.com
```

Comment:
I'd recommend replacing `dummy.dummy.com` with `kong.example`, since `.example` is a designated documentation TLD. `dummy.dummy.com` does actually resolve (it appears to be a stub site used to redirect to an Amazon affiliate link).

Reply:
sorry I did not check your comment, I will update the PR as soon as possible